include/onnxruntime/core/graph/graph.h

```diff
@@ -469,6 +477,10 @@ class Node {
 
   // OperatorSchema that <*this> node refers to.
   const ONNX_NAMESPACE::OpSchema* op_ = nullptr;
+
+  // Execution priority, lower value for higher priority
+  int priority_ = 0;
```

Comment:
Whether we should have a uniformed way to define priority_ ? e.g., some sort of ENUM. Otherwise, different users may be priority set in conflict right?

Reply:
Yes. But still need some handle tuning to get a better understand of the recomptue's behavior when we have multiple goals for setting the execution order. 
Created a placeholder for now, will update in latter PRs.